spec/features/course/unread_status_management_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'Announcements read/unread status management', type: :feature do
+
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+
+    let(:user) { create(:user, role: :administrator) }
+    let(:course) { create(:course) }
+
+    before do
+      login_as(user, scope: :user)
+    end
+
+    context 'there is a new announcement' do
```

Comment:
Check here for writing better specs: http://betterspecs.org

Most of your description messages are not very good.